### DISCUSSION TITLE
formatEther precision

### DISCUSSION THREAD

**Discussion by pociej** [wevm/viem#1341] [General]
As it is perfectly common not display balance with wei precision `formatEther` could have second parameter `precision`  

```
formatEther({ wei: 69631459869036871033, precision : 4} )  = 69.6315
 //or formatEther(69631459869036871033,4) but im fan of naming params 
```

**Comment by tmm** [comment]
Check out [dnum](https://github.com/bpierre/dnum) for now.

**Comment by pociej** [comment]
@tmm thx. Yeah, you known  this thing is small like few lines of code and I can have plenty of option including own implementation Just wanted to leave suggestion.

**Reply by tmm** [reply]
Appreciate the feedback!
